charts/backingservices/charts/srs/.helmignore

```diff
@@ -0,0 +1,22 @@
+# Patterns to ignore when building packages.
+# This supports shell glob matching, relative path matching, and
+# negation (prefixed with !). Only one pattern per line.
+.DS_Store
+# Common VCS dirs
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
+*.tmp
+*~
+# Various IDEs
+.project
+.idea/
+*.tmproj
+.vscode/
```

Comment:
AFAIK, .helmignore is applicable to the individual chart or subchart directory it is located in, I dont know whether having it on the main chart is applied to subcharts or not, I could not find enough documentation around this, I referred to some other charts on github here https://github.com/elastic/helm-charts where subcharts themselves define the .helmignore, please confirm if this can be removed safely?